### DISCUSSION TITLE
对仓库pr合并的小小建议

### DISCUSSION THREAD

**Discussion by xuanxuan1231** [Class-Widgets/Class-Widgets#187] [建议]
RT，作者目前合并pr大概是用的第一种默认方法
然而这样会在在提交记录中显示很多commit，包括pr中的commit和合并的commit
所以说个人建议作者试试squash and merge，这样可以把整个pr合并成一个commit，也不会出现多的合并的commit

**Comment by RinLit-233-shiroko** [comment]
彳亍
